terraform/trial_rg/services.tf

```
@@ -100,6 +111,13 @@ module "trial_sc_config" {
   docker_image        = var.trial_sc_config_image_name
   docker_image_tag    = var.trial_sc_config_image_tag
 
+  settings = {
+    "SPRING_CLOUD_CONFIG_SERVER_GIT_URI"         = var.trial_sc_config_git_uri,
+    "SPRING_CLOUD_CONFIG_SERVER_GIT_SEARCHPATHS" = var.trial_sc_config_search_paths
+    "SERVER_PORT"                                = 8080
```

I think it should be port 8888 here, wdyt
```suggestion
    "SERVER_PORT"                                = 8888
```

---

8888 was used for local run.
the sc config server Dockerfile exposes 8080 and also Line 118 exposes web app 8080 port.